aries-site/src/components/feedback/Question.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormField,
+  TextArea,
+  Text,
+  RadioButtonGroup,
+  FileInput,
+} from 'grommet';
+import { StarRating } from './StarRating';
+import { ThumbsRating } from './ThumbsRating';
+
+export const Question = ({ formProps, label, inputProps, kind, name }) => {
+  const accessibilityProps = {
+    id: name,
+    name,
+  };
+
+  const formats = {
+    star: { render: StarRating, border: false },
+    thumbs: {
+      render: ThumbsRating,
+      border: false,
+      label: <Text size="large">{label}</Text>,
```

Comment:
Do we always want the large size for this? I thought we had settled on default formfield styling?

Reply:
yeah we went back and forth but from this comment from Matt.. Megan also felt the same
```
The question text is feeling small and difficult to read. I'm thinking that the feedback context is different than a true form context, and the text is having to do a lot more communicating and heavy lifting.
```

Reply:
I think the customer/user feedback context is different than an enterprise form context and is deserving of a different treatment. The readability of a lengthy question string is much different than a short input label.

This is also a prototype, so we are not committing at this point. But if we were, I'd argue for better readability considering the context.